.travis.yml

```diff
@@ -1,6 +1,5 @@
 language: ruby
 rvm:
-    - 1.9.2
```

Comment:
The parallel gem required by cane won't build in 1.9.2. Not truly breaking, but we also haven't supported 1.9.2 in the downstreams for a while.